playbook/app/views/playbook/samples/samples_index.html.erb

```diff
@@ -13,46 +13,55 @@
   <% end %>
 
   <%= pb_rails("background", props: { background_color: "white", padding: "xl" }) do %>
+    <div class="main-content-wrapper">
+      <%# Loop through each Sample Category in samples.yml %>
+      <% @sample_yaml.each_with_index do |category, i| %>
 
-    <%# Loop through each Sample Category in samples.yml %>
-    <% @sample_yaml.each_with_index do |category, i| %>
+        <% category_title = category[0] %>
+        <% category_pages = @sample_yaml[category_title] %>
 
-      <% category_title = category[0] %>
-      <% category_pages = @sample_yaml[category_title] %>
+        <% if category_pages %>
 
-      <% if category_pages %>
-        <%= pb_rails("flex", props: {
-          classname: "full_page samples_flex_wrapper",
-          horizontal: "center",
-          margin_top: i > 0 ? "xl" : "",
-          wrap: true
-        }) do %>
+          <% if i > 0 %>
+            <%= pb_rails("section_separator", props: { margin_top: "xl" }) %>
+          <% end %>
 
-          <%= pb_rails("flex/flex_item", props: { classname: "preview" }) do %>
-            <%= pb_rails("title", props: { margin_bottom: "sm", text: category_title.titleize, tag: "h4", size: 4 }) %>
-            <%= image_tag("#{category_title}.svg", class: "sample_screenshot") %>
+          <%= pb_rails("flex", props: { margin_bottom: "md", margin_top: i > 0 ? "xl" : "", vertical: "center" }) do %>
+            <%= pb_rails("title", props: { text: category_title.titleize, tag: "h2", size: 2 }) %>
             <%= pb_rails("icon_value", props: {
               icon: "palette",
-              margin_top: "xs",
+              margin_left: "xs",
               text: "#{category_pages.size} Designs"
             }) %>
           <% end %>
-          
-          <%= pb_rails("flex/flex_item", props: { classname: "list" }) do %>
-            <%= pb_rails("nav", props: {}) do %>
+
+          <%= pb_rails("layout", props: { layout: "collection" }) do %>
+            <%= pb_rails("layout/body", props: { classname: "sample-card-grid-override" }) do %>
               <% category_pages.each do |link| %>
-                <%= pb_rails("nav/item", props: { 
-                  icon_right: "angle-right",
-                  text: link == "crm_client_list" ? "CRM Client List" : link.titleize,
-                  link: "/samples/#{link}" 
-                }) %>
+                <%= pb_rails("card", props: { classname: "ui-sample-card", padding: "md" }) do %>
+                  <a href=<%="/samples/#{link}"%>>
```

Comment:
The click state is very small. I expected the whole card to be clickable. Can you wrap the card or expand the click state?